### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Manage Access Policies_Manage Access Policies_HeadingAndLabels:Access Policies looks like a heading but it is not defined using any heading level 

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5723]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev3; #BM_Azure Tools Storage Explorer_May2022; #Azure Tools Storage Explorer; #WCAG2.4.6; #Headings and Labels; #FTP; 

## Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Create Blob Container" and hit enter.
5. Right Click on newly created blob container and select "Manage Access Policies" and hit enter.
6. Verify that the text "Access Policies" is MAS Compliant.

## Actual results:
"Access Policies" text present in "Manage Access Policies" window looks like heading, but it is not defined in any heading level.

## Expected results:
"Access Policies" text should be defined using H1 level heading.

## User Impact:
Users might get confused if "Access Policies" text is not defined using any heading level.

## Attachments:
![5723_A11y_AzureToolStorage_ManageAccessPolicies_HeadingLevel](https://user-images.githubusercontent.com/105198219/168268127-68243f08-b267-4e08-bdbb-88e99a1aff38.png)

**Comment by MRayermannMSFT** [comment]
Hi @ManishaC1 

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-ia32.exe

**Comment by ManishaC1** [comment]
Issue is resolved in the provided environment(ver 1.25.0). Hence closing the bug.
![5723_Issue not repro](https://user-images.githubusercontent.com/105198219/175864059-a5e101c7-77f5-4651-b430-6aabd37c24b7.png)
